### PR TITLE
initial update

### PR DESCRIPTION
**Pull request by maximusmith529**
## Overall Description:

added DISPLAYNAME, symbolizing the user's chosen display name. Can be null as it isn't set by anything prior.

## What Was Changed:

create_storyforge_db.sql

## Any Notes For Testing:

I cannot test this yet.

## Link To Ticket:

[Link to the ticket this pr is for](https://trello.com/c/KGSQYSAU/39-update-user-info-table-to-use-display-name/)

### CODE REVIEW

File: create_storyforge_db.sql [hunk 1]
```diff
@@ -29,6 +29,7 @@ CREATE TABLE USER_LOGIN (
 -- Create USER_INFO Table
 CREATE TABLE USER_INFO (
     USER_ID INT,
+    DISPLAYNAME VARCHAR(255),
     EMAIL VARCHAR(255) NOT NULL,
```
Comment (sethlenhof) [inline create_storyforge_db.sql:33]: can u delete this no longer needed

Comment (sethlenhof) [reply create_storyforge_db.sql:33]: for email in that table

Comment (maximusmith529) [reply create_storyforge_db.sql:33]: gotcha

**Review by sethlenhof — APPROVED** [review]
Display name!

**Pull request merged**